dashboard/app/controllers/musiclab_controller.rb

```diff
@@ -1,12 +1,38 @@
 class MusiclabController < ApplicationController
   ANALYTICS_KEY = CDO.amplitude_api_key
+  PROJECT_BEATS_LEVEL_NAME = 'projectbeats'
 
   def index
     view_options(no_footer: true, full_width: true)
     @body_classes = "music-black"
 
     projects = Projects.new(get_storage_id)
-    @channel_id = projects.most_recent_project_type('music') || ChannelToken.create_channel(request.ip, projects, type: 'music')
+
+    # Find the project ID (channel ID) for the /projectbeats Music Lab project. If there already
+    # is a channel ID with the 'projectbeats' level name, then use that. If not, find the most
+    # recent Music Lab project. If this project is standalone, it should be the existing
+    # /projectbeats project. Create a new project with the 'projectbeats' level name, and copy
+    # source code from the existing project so that users don't lose their code.
+    project_beats_id = projects.most_recent(PROJECT_BEATS_LEVEL_NAME, include_hidden: true)
+    if project_beats_id
+      @channel_id = project_beats_id
+    else
+      last_music_project_id = projects.most_recent_project_type('music')
+
+      new_project_data = {
+        name: 'New Project Beats Project',
```

Comment:
Should we loc this at some point?

Reply:
We can, but it currently doesn't show up anywhere that's visible to the user, since this project is hidden from their gallery.